### PR TITLE
Add correct collectd dir for rpm systems

### PR DESCRIPTION
**Pull request by sugr4**
@Microsoft/omsagent-devs

### CODE REVIEW

File: installer/bundle/bundle_skel.sh [hunk 1]
```diff
@@ -550,10 +549,10 @@ do
             ;;
 
         --collectd)
-            if [ -d "${COLLECTD_DIR}" ]; then
+            if [ -f /etc/collectd.conf ] || [ -f /etc/collectd/collectd.conf ]; then
```
Comment (jeffaco) [inline installer/bundle/bundle_skel.sh]: In shell, that's not the way you do that.

This should be something like:

`if [ -f /etc/collectd.conf  -o -f /etc/collectd/collectd.conf ]; then`

I'm not even sure the || construct will work reliably, as that's a pipe thing in this context.

File: installer/bundle/bundle_skel.sh [hunk 2]
```diff
@@ -681,8 +680,11 @@ if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
             rmdir /etc/opt /var/opt > /dev/null 2> /dev/null || true
         fi
     fi
-    if [ -f ${COLLECTD_DIR}/oms.conf ]; then
-        rm ${COLLECTD_DIR}/oms.conf > /dev/null 2> /dev/null
+    if [ -f /etc/collectd.d/oms.conf ]; then
+        rm /etc/collectd.d/oms.conf > /dev/null 2> /dev/null
+    fi
+    if [ -f /etc/collectd/collectd.conf.d/oms.conf ]; then
+        rm /etc/collectd/collectd.conf.d/oms.conf > /dev/null 2> /dev/null
```
Comment (jeffaco) [inline installer/bundle/bundle_skel.sh]: You could just replace all this with two lines:

```
rm -f /etc/collectd.d/oms.conf > /dev/null 2> /dev/null
rm -f /etc/collectd/collectd.conf.d/oms.conf > /dev/null 2> /dev/null
```

The -f says force remove, and don't consider it an error if the file doesn't exist. That eliminates the pre-checks.

**Comment by jeffaco** [comment]
Fix the "or" handling in the shell scripting, otherwise looks fine to me, signed off.

**Comment by sugr4** [comment]
Made changes based on comments, built an tested new bundles.

**Pull request merged**